src/test/java/net/snowflake/client/RunningNotOnJava21.java

```diff
@@ -0,0 +1,13 @@
+package net.snowflake.client;
+
+import static net.snowflake.client.jdbc.SnowflakeUtil.systemGetProperty;
+
+public class RunningNotOnJava21 implements ConditionalIgnoreRule.IgnoreCondition {
```

Comment:
isn't that a blocker? will it work for next Java versions?

Reply:
certificate generation needs to be implemented for next Java versions (also for 21). As it's a jar, we can always install i.e. jre 17 and start wiremock using that java version